testing/test_assertion.py

```diff
@@ -981,6 +982,30 @@ class SimpleDataObjectTwo:
         assert lines is None
 
 
+class TestAssert_reprcompare_namedtuple:
+    def test_namedtuple(self) -> None:
+        SimpleDataObject = namedtuple("SimpleDataObject", ["field_a", "field_b"])
```

Comment:
since `collections` is already imported this could be `collections.namedtuple`

Reply:
`collections.abc` is imported. I don't see collections being imported. Are you suggesting to just do `import collections` and use `collections.namedtuple`?

Reply:
`import collections.abc` is equivalent to
```
importlib.import_module("collections.abc")
import collections
```

Reply:
TIL, I mostly prefer it to be explicit. Even on `import collections` the pre-commit removes it. So I have made the changes as suggested.

Reply:
the hook removes it because `import a` and `import a.b.c` are duplicate imports